tests/test_alembic.py

```diff
@@ -36,6 +36,9 @@ def test_alembic(app, database):
     tables = [x.name for x in db.get_tables_for_bind()]
     assert 'vocabularies_metadata' in tables
     assert 'vocabularies_types' in tables
+    assert 'vocabularies_subtypes' in tables
+    # NOTE: The below was created 5 months ago and context unclear, so
```

Comment:
These are remainings from works on subjects that got stopped. Once affiliations are done, and we start with subjects, this part will be re-done. 

Major: I learnt it not long ago the same way, but we should **not use NOTE** and just leave it as a normal comment if it's not a TODO/FIXME because this keyword (NOTE) has a specific meaning for translations and will be picked up and appear in translation files 😅 I imagine the confusion of translators hehe

I've been removing them when I see them, but I used them quite extensively, let's try to remove them.

[Reference](http://babel.pocoo.org/en/latest/messages.html#translator-comments)

Reply:
Ha :smile: ! Good thing to know about `NOTE` . I will remove.